### PR TITLE
Feature/add toro tapas menu

### PR DESCRIPTION
**Pull request by antonio06**
@manudous

### CODE REVIEW

File: backend/src/console-runners/restaurant-list/toro-tapas.ts [hunk 1]
```diff
@@ -0,0 +1,724 @@
...
+            rationsTypes: [{ unit: 'Media', price: 1.7 }],
+          },
+        },
+        {
+          name: 'Taboulé',
+          priceByRation: {
+            rationName: 'Tapas',
+            rationsTypes: [
+              { unit: 'Media', price: 1.7 },
+              { unit: 'Entera', price: 5 },
```
Comment (manudous) [inline backend/src/console-runners/restaurant-list/toro-tapas.ts:33]: unit: 'Ración'

File: backend/src/console-runners/restaurant-list/toro-tapas.ts [hunk 2]
```diff
@@ -0,0 +1,724 @@
...
+            rationsTypes: [{ unit: 'Media', price: 1.7 }],
+          },
+        },
+        {
+          name: 'Hummus',
+          priceByRation: {
+            rationName: 'Tapas',
+            rationsTypes: [
+              { unit: 'Media', price: 2 },
+              { unit: 'Entera', price: 6 },
```
Comment (manudous) [inline backend/src/console-runners/restaurant-list/toro-tapas.ts:50]: same as before and for the rest of the tapas-raciones page

File: backend/src/console-runners/restaurant-list/toro-tapas.ts [hunk 3]
```diff
@@ -0,0 +1,724 @@
...
+  menuDate: new Date('2022-04-04'),
+  theme: 'tapas',
+  menu: [
+    {
+      name: 'Tapas Raciones',
+      items: [
+        {
+          name: 'Gazpacho',
+          priceByRation: {
+            rationName: 'Tapas',
```
[the remark this answers is not in the document]
Comment (antonio06) [reply backend/src/console-runners/restaurant-list/toro-tapas.ts:23]: done

File: backend/src/console-runners/restaurant-list/toro-tapas.ts [hunk 4]
```diff
@@ -0,0 +1,724 @@
...
+            rationsTypes: [{ unit: 'Media', price: 1.7 }],
+          },
+        },
+        {
+          name: 'Taboulé',
+          priceByRation: {
+            rationName: 'Tapas',
+            rationsTypes: [
+              { unit: 'Media', price: 1.7 },
```
[the remark this answers is not in the document]
Comment (antonio06) [reply backend/src/console-runners/restaurant-list/toro-tapas.ts:32]: done

File: backend/src/console-runners/restaurant-list/toro-tapas.ts (hunk 2 above)
Comment (antonio06) [reply backend/src/console-runners/restaurant-list/toro-tapas.ts]: done

**Pull request merged**